yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/check/advanced/RequestedTypeListSources.java

```diff
@@ -0,0 +1,35 @@
+package com.yoti.api.client.docs.session.create.check.advanced;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RequestedTypeListSources extends RequestedCaSources {
+
+    @JsonProperty("types")
+    private final List<String> types;
+
+    RequestedTypeListSources(List<String> types) {
+        this.types = types;
+    }
+
+    public List<String> getTypes() {
+        return types;
+    }
+
+    public static class Builder {
```

Comment:
Do we want a _builder()_ method on this class too?

And the others that extend some abstract base class with the builders on it